### PR TITLE
fix some broken url

### PR DESCRIPTION
**Pull request by tanjunchen**
fix-some-broken url

### CODE REVIEW

**Comment by tanjunchen** [comment]
@googlebot I signed it!

File: content/en/news/2019/istio-security-2019-002/index.md [hunk 1]
```diff
@@ -110,4 +110,4 @@ The setup script uses helm template to produce an `envoyFilter` resource that de
 
 The Istio team would like to thank Divya Raj for the original bug report.
 
-We’d like to remind our community to follow the vulnerability reporting process described at [https://istio.io/about/security-vulnerabilities/](/about/security-vulnerabilities/) to report any bug that can result in a security vulnerability
+We’d like to remind our community to follow the vulnerability reporting process described at [these procedures](https://istio.io/about/security-vulnerabilities/) to report any bug that can result in a security vulnerability
```
Comment (sdake) [inline content/en/news/2019/istio-security-2019-002/index.md]: if you can revert this change, i'd be happy to merge.  I want to keep the URL intact so contributors have more awareness of the secvn proces.

Comment (tanjunchen) [reply content/en/news/2019/istio-security-2019-002/index.md]: OK Thanks

**Comment by tanjunchen** [comment]
@sdake done
